coffee/utils.py

```diff
@@ -451,6 +480,33 @@ def find_expression(node, type=None, dims=None, in_syms=None, out_syms=None):
     return exprs
 
 
+def summands(node):
+    """
+    Return the top-level summands in /node/.
+
+    Examples
+    ========
+
+    a + b --> [a, b]
+    a*b*c --> [a*b*c]
+    a*b*c + c*d --> [a*b*c, c*d]
+    (a+b)*c + d --> [(a+b)*c, d]
+    foo(a) --> []
+    """
+
+    handle = zip(*explore_operator(node))
```

Comment:
Suggested fix: cast iterable to list. The iterable is always "true" in a condition, whether it will yield any elements or not.